src/main/java/uk/nhs/hee/tis/revalidation/connection/message/RabbitMessageListener.java

```diff
@@ -88,7 +88,7 @@ public void receiveMessageGetMaster(final String getMaster) {
       //Delete and create elastic search index
       elasticSearchIndexUpdateHelper.clearConnectionIndexes(ES_INDICES);
 
-      final List<ConnectionInfoDto> masterList = masterElasticSearchService.findAllScroll();
+      final List<ConnectionInfoDto> masterList = masterElasticSearchService.findAllBySpringDataScroll();
```

Comment:
The original name was fine, we wouldn't want to have to rename the method if we ever changed how it was implemented.

Reply:
Method name changed to "findAllScroll". Thanks